app/views/Dashboard/Country/styles.css

```diff
@@ -30,20 +29,61 @@
                 border-radius: var(--dui-border-radius-card);
 
                 .responsive-content {
-                    min-height: 20rem;
+                    min-height: 32rem;
                 }
             }
         }
 
         .country-info {
-            flex-basis: 25%;
-            flex-shrink: 0;
             border: var(--width-separator-thin) solid var(--color-separator-light);
             border-radius: var(--dui-border-radius-card);
             background-color: var(--dui-color-foreground);
-        }
+            min-width: 20rem;
```

Comment:
Let's use variable rather than absolute values for widths.

Reply:
changes done